### PR TITLE
Adding logs

### PR DESCRIPTION
**Pull request by andrea-pasquale**
This PR adds some logs in order to show which protocol is being executed.
The timing of both the acquisition and the fitting are shown for each protocol.
Here is an example

image

Checklist:
- [ ] Reviewers confirm new code works as expected.
- [x] Tests are passing.
- [x] Coverage does not decrease.
- [x] Documentation is updated.

### CODE REVIEW

**Comment by Jacfomg** [comment]
Usually the fit takes almost nothing, could we add a check to only log that time when is above a threshold (1s maybe) ?

**Comment by rodolfocarobene** [comment]
I kinda liked the "timing.json" file that we are producing for the benchmarks... Now that the the process is nicely written do you think we could have it directly in qibocal or is it useless?

**Comment by andrea-pasquale** [comment]
> Usually the fit takes almost nothing, could we add a check to only log that time when is above a threshold (1s maybe) ?

Yep, I can put a threshold. Maybe 1 second is too much (?)

**Comment by andrea-pasquale** [comment]
> I kinda liked the "timing.json" file that we are producing for the benchmarks... Now that the the process is nicely written do you think we could have it directly in qibocal or is it useless?

I need to see if there is a way to retrieve the information from the decorator. I feel like we don't need another json.
Perhaps we could store it somewhere else.

File: src/qibocal/auto/operation.py [hunk 1]
```diff
@@ -24,6 +28,25 @@
 """Name of the file where results are dumped."""
 
 
+def show_logs(func):
+    """Decorator to add logs."""
+
```
Comment (rodolfocarobene) [inline src/qibocal/auto/operation.py:33]:
```suggestion
```

**Review by rodolfocarobene — APPROVED** [review]

File: src/qibocal/auto/operation.py (hunk 1 above)
Comment (andrea-pasquale) [reply src/qibocal/auto/operation.py:33]: Thanks for catching this.

**Comment by andrea-pasquale** [comment]
Just to a give an update now the timings are stored in the `meta.json` file (formerly it was a `yaml` but since it should not be modified by the user I decided to converted it to json.)
I've also added a threshold as requested .
Thanks @rodolfocarobene for the review already.

**Review by Jacfomg — APPROVED** [review]
Great thanks!

**Pull request merged**